### ISSUE TITLE
Staging Review finding: Imposter buttons

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#126935]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** govcio-vebt
**Product:** Comparison Tool
**Feature:** Student feedback grid

## Findings details
**VA.gov Experience Standard - issue:** [User encounters a component or pattern that isn't considered a best practice.](https://design.va.gov/about/experience-standards/#user-encounters-components-and-patterns-that-meet-best-practices)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Imposter component:** [Yes](https://design.va.gov/components/#governance-rules-for-using-components)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @allison0034 (Design), @jasonday (Accessibility)

### Description

There are 24 "learn more" buttons on the page that are imposter components. They are styled to look like links.

<details><summary>Link, screenshot or steps to recreate</summary>

![learn](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/110858-allison-5.png?raw=true)

</details>

## Recommended action

Since the action is to open a modal, it must be a button for accessibility reason. Use the VADS button component.

https://design.va.gov/components/button/#choose-the-right-element-buttons-vs-links

## References

- **Accessibility Defect Severity:** N/A
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** [Button component](https://design.va.gov/components/button)
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoez) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
